processing/src/main/java/io/druid/query/groupby/epinephelinae/RowBasedGrouperHelper.java

```diff
@@ -497,14 +491,14 @@ public Comparable get()
     @Override
     public Supplier<Comparable> makeInputRawSupplier(FloatColumnSelector selector)
     {
-      return new Supplier<Comparable>()
-      {
-        @Override
-        public Comparable get()
-        {
-          return selector.get();
-        }
-      };
+      return () -> selector.get();
+    }
+  }
+  private static class DoubleInputRawSupplierColumnSelectorStrategy implements InputRawSupplierColumnSelectorStrategy<DoubleColumnSelector> {
```

Comment:
DoubleInputRawSupplierColumnSelectorStrategy could be a singleton lambda itself, and switch below always return the same instance